e2e-tests/upgrade/run

```diff
@@ -102,8 +102,11 @@ main() {
 	IMAGE_BACKREST="${TARGET_IMAGE_BACKREST:-"perconalab/percona-postgresql-operator:main-ppg$PG_VER-pgbackrest"}"
 	IMAGE_BACKREST_REPO="${TARGET_IMAGE_BACKREST_REPO:-"perconalab/percona-postgresql-operator:main-ppg$PG_VER-pgbackrest-repo"}"
 	IMAGE_PGBADGER="${TARGET_IMAGE_PGBADGER:-"perconalab/percona-postgresql-operator:main-ppg$PG_VER-pgbadger"}"
+    desc 'remove operator'
+	kubectl_bin delete deployment/postgres-operator
 
-	deploy_operator '' '' 'update'
+	sleep 10
+	deploy_operator
```

Comment:
should we ditch arguments for deploy_operator here?

Reply:
We had an error in deploy_operator func, fixed in https://github.com/percona/percona-postgresql-operator/pull/183/commits/38c2a769adc31480d6aa072ff8a5ccb40e8722c5